### DISCUSSION TITLE
How to pip install for Python 3.10 under Windows 11

### DISCUSSION THREAD

**Discussion by yportier** [Toblerity/Fiona#1154] [Q&A] [answered]
The Windows section of [Fiona's pypi](https://pypi.org/project/Fiona/) still recommends getting binary installers from https://www.lfd.uci.edu/~gohlke/pythonlibs/#fiona and GDAL distribution from http://www.gisinternals.com/release.php depsite C. Gohlke's site being archived since June and GisInternals not offering distributions for Python > 3.9.  
What is the way forward ?

**Comment by sgillies** [comment, marked as answer]
There are new Windows wheels on PyPI today: https://pypi.org/project/Fiona/1.8.22/#files. `pip install fiona` might work for you. The Anaconda package system and conda-forge package channel have been, and remain, a very good option for Windows users.

**Reply by yportier** [reply]
Perfect, that did the trick, many thanks!
I understand this is well outside Fiona's scope, but would you know of any source for GDAL binaries aimed at Python 3.10 ?
